crates/nu-command/src/commands/core_commands/source.rs

```diff
@@ -69,11 +69,15 @@ pub fn source(args: CommandArgs) -> Result<OutputStream, ShellError> {
         for lib_path in dir {
             match lib_path {
                 Ok(name) => {
-                    let path = PathBuf::from(name).join(source_file);
-
-                    if let Ok(contents) =
-                        std::fs::read_to_string(&expand_path(Cow::Borrowed(path.as_path())))
-                    {
```

Comment:
This shouldn't error since this part checks to see if the file is under the `lib_dirs` optional search paths (where you'd have your nu libraries and the like)